host/xtest/pkcs11_1000.c

```diff
@@ -1580,6 +1588,29 @@ static void xtest_pkcs11_test_1007(ADBG_Case_t *c)
 ADBG_CASE_DEFINE(pkcs11, 1007, xtest_pkcs11_test_1007,
 		"PKCS11: Check operations release at session closure");
 
+#define CK_MAC_KEY_AES(_key_array) \
+	{								\
+		{ CKA_SIGN,	&(CK_BBOOL){CK_TRUE},			\
+				sizeof(CK_BBOOL) },			\
+		{ CKA_VERIFY,	&(CK_BBOOL){CK_TRUE},			\
+				sizeof(CK_BBOOL) },			\
+		{ CKA_CLASS,	&(CK_OBJECT_CLASS){CKO_SECRET_KEY},	\
+				sizeof(CK_OBJECT_CLASS) },		\
+		{ CKA_KEY_TYPE,	&(CK_KEY_TYPE){CKK_AES},		\
+				sizeof(CK_KEY_TYPE) },			\
+		{ CKA_VALUE,	(void *)_key_array,			\
+				sizeof(_key_array) }			\
```

Comment:
nitpick: prefer with a coma `,` at end of line.

Reply:
Ok. Was trying to be consistent with `CK_MAC_KEY_HMAC`. Will add a comma there as well.